ToDoApp/ToDoApp/ToDoListView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ToDoListView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-07.
+//
+
+import SwiftUI
+
+struct ToDoSection: Identifiable {
+    var id: UUID = UUID()
+    let title: String
+    let toDoItems: [TaskModel]
+}
+struct ToDoListView: View {
+    static let items: [ToDoSection] = [
+            .init(title: "Today’s To-Do List", toDoItems: [
+                .init(title: "Cleaning", deadline: Date(), status: false),
+                .init(title: "Cooking", deadline: Date(), status: true),
+                .init(title: "Painting", deadline: Date(), status: true),
+                .init(title: "Learning", deadline: Date(), status: true),
+                .init(title: "Midterm", deadline: Date(), status: true),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Clean car", deadline: Date(), status: true),
+                .init(title: "Workout", deadline: Date(), status: true),
+                .init(title: "Cooking", deadline: Date(), status: false)
+                
+            ]),
+            .init(title: "All To-Do List", toDoItems: [
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Cleaning", deadline: Date(), status: true),
+                .init(title: "Final", deadline: Date(), status: false),
+                .init(title: "Grocery", deadline: Date(), status: true),
+                .init(title: "Dancing", deadline: Date(), status: false),
+                .init(title: "Workout", deadline: Date(), status: false),
+                .init(title: "Laundry", deadline: Date(), status: false),
+                .init(title: "Drawing", deadline: Date(), status: false),
+                .init(title: "Painting", deadline: Date(), status: true)
+            ])
+        ]
+    var body: some View {
+        NavigationStack{
+            List {
+                ForEach(ToDoListView.items) { tasks in
+                    HStack(spacing: 6) {
+                        Text(tasks.title)
+                            .foregroundColor(Color(.caption))
+                            .font(.caption)
```

Comment:
just a hint, when there are the same modifiers applied to child views inside a container view, as an option you can apply this modifier only once to the parent container view and SwiftUI will pass this setting to all of the child views. it will look like this:
instead of 
```
HStack {
    Text(title)
        .font(.caption)
    Text(subtitle)
        .font(.caption)
}
```
it can also be achieved in that way:
```
HStack {
    Text(title)
    Text(subtitle)
}
.font(.caption)
```